tests/parsers/test_interop_parser.py

```diff
@@ -31,77 +32,94 @@ def subscribe(self):
                     self.percent_q30_values.append(interop_stat)
                 if key == "percent_q30_per_cycle":
                     self.percent_q30_per_cycle.append(interop_stat)
+                if key == "percent_phix":
+                    self.phix_aligned_values.append(interop_stat)
```

Comment:
As it is now, only one of these statements will be executed each time, but one needs to carefully look at all the conditions to see that it's the case. This could be rewritten either using a `match` statement, or (even better) by lumping all the attributes defined in `__init__` into a dictionary using the same key that is found in `interop_stat`.

Reply:
Very good point, I will address this.